src/esp/bindings_js/bindings_js.cpp

```diff
@@ -238,6 +239,23 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
       .property("name", &ActionSpec::name)
       .property("actuation", &ActionSpec::actuation);
 
+  em::class_<Ray>("Ray")
+      .constructor<>()
+      .constructor<Magnum::Vector3, Magnum::Vector3>();
+
+  em::class_<RayHitInfo>("RayHitInfo")
+      .property("objectId", &RayHitInfo::objectId)
+      .property("point", &RayHitInfo::point)
+      .property("normal", &RayHitInfo::normal)
+      .property("rayDistance", &RayHitInfo::rayDistance);
+
+  em::class_<RaycastResults>("RaycastResults")
+      .smart_ptr_constructor("RaycastResults", &RaycastResults::create<>)
+      .function("hasHits", &RaycastResults::hasHits)
+      .function("sortByDistance", &RaycastResults::sortByDistance)
```

Comment:
Probably don't need to expose this. It will be called internally by `Simulator::castRay` which you are also binding.

Reply:
I believe this is actually necessary? It doesn't build if I don't include this binding because I am doing stuff with `RaycastResults` in `test_page.js`.

Reply:
Yeah, @aclegg3 it's necessary to be bound for castRay to return it.

Reply:
Sorry, github expanded my comment. I meant specifically `sortByDistance` which is only used inside `Simulator::castRay`.

Reply:
Ah ok then that makes sense; sure I can remove it if it's not useful.